log/bench_test.go

```diff
@@ -63,6 +63,10 @@ func BenchmarkLoggers(b *testing.B) {
 			name:    "eight values",
 			keyVals: []any{"one", 1, "two", 2, "three", 3, "four", 4, "five", 5, "six", 6, "seven", 7, "eight", 8},
 		},
+		{
```

Comment:
aint this just a duplicate from the above case ?

Reply:
Yes 😅. Probably airdrop farming...